bin/grocery-delivery

```diff
@@ -120,7 +127,12 @@ def partial_upload(knife, repo, checkpoint, local_head)
   end
 
   knife.cookbook_delete(deleted_cookbooks) if deleted_cookbooks
```

Comment:
is this still the right place to do deletes - or do you need to tell berks you want to delete to handle any dependencies?